server/lib/pbench/__init__.py

```diff
@@ -229,6 +250,7 @@ def mocked_time():
                 return 0
             global _time
             _time = mocked_time
+            #self.logger_type = "file"
```

Comment:
Can you remove this line entirely?  Commented-out lines with out comments that describe why it is there can end up reducing the clarity of the code when somebody else tries to read and understand what it does.